tools/generator/src/DTO/XcodeScheme.swift

```diff
@@ -132,11 +134,21 @@ extension XcodeScheme {
 }
 
 extension XcodeScheme {
+    struct PrePostAction: Equatable, Decodable {
+        let name: String
+        let target: BazelLabel
+        let scriptContents: String
+    }
+    
```

Comment:
Since this will apply to more than just the `BuildAction` in the future, can you extract it to it's own `extension` above this one?